Library/Homebrew/system_command.rb

```diff
@@ -70,7 +70,7 @@ def run!
     params(
       executable:   T.any(String, Pathname),
       args:         T::Array[T.any(String, Integer, Float, URI::Generic)],
-      sudo:         T::Boolean,
+      sudo:         T.any(T::Boolean, String),
```

Comment:
I think this is pretty weird. I'd rather a new `sudo_user` argument or similar.

Reply:
Should `sudo: false, sudo_user: 'root'` throw an error in this case?

Reply:
Yes, I think so.